registration-processor-default.properties

```diff
@@ -987,17 +979,54 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
-#updated properties name with the prefix (mosip.regproc)
+
+
 mosip.regproc.landing.zone.account.name=landing-zone
 mosip.regproc.landing.zone.type=ObjectStore
-
 mosip.regproc.landing.zone.fixed.delay.millisecs=120000
 mosip.regproc.landing.zone.inital.delay.millisecs=120000
 
 registration.processor.lostrid.max.registrationid=5
 mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
+
 ##Ability to update contact information as part of LOST UIN
-uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
+uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress,addressLine1,addressLine2,addressLine3
+
+CREDENTIALREQUESTV2=${mosip.idrepo.credrequest.generator.url}/v1/credentialrequest/v2/requestgenerator
+
+mosip.registration.processor.digitalcard.credentialtype=PDFCard
+mosip.registration.processor.print.credentialtype=euin
+mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
 
 ##timeout in milliseconds for health check registrer
-mosip.regproc.health-check.handler-timeout=2000
\ No newline at end of file
+mosip.regproc.health-check.handler-timeout=2000
+
+# this property is used to define issuers eg partnerid:credentialType:templateTypeCode
+mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
+mosip.registration.processor.print.issuer.noMatch=mpartner-default-print#vercred#RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.print.issuer.config-map={:}
+# This property is used to define default internal credential issuers separated by comma.
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.internal.issuers=mpartner-default-digitalcard#PDFCard#RPR_UIN_CARD_TEMPLATE
+# This property is used to define default issuers separated by comma,
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.issuers=mpartner-default-print#euin#RPR_UIN_CARD_TEMPLATE
+# Credential Partner Configuration
+# Map all print partners for a respective attribute ('region') value (any appropriate valid id-schema field) in the format given below,
+# Provide attribute('region') value in the first language of mandatory languages(mosip.mandatory-languages) defined.
+# The config map supports configuring multiple print partners separated by comma(',').
+# The partner can be configured as key and value pair where key would be string value (partnerId#credential_type#template_code) and value would be an expression
+# ({list_of_attribute_values} contains attribute_field), here 'mpartner-default-print' partner used as an example.
+mosip.identification.attribute=region
```

Comment:
do we need the above new lines ? These all not moved into registration-processor-credential-partners.json ?

Reply:
we can remove these lines as we are not using any of the property in credential requestor stage.

Reply:
@Khuddusshariff0022 and @ashok-ksharma to work with @dhanendra06 to remove the unwanted properties

Reply:
Some of the properties are moved to bootstrap.Properties. which can be removed and some are not in use that also can be removed.

Reply:
Done

Reply:
I see few other properties like 
registration.processor.reprocess.restart-from-stage and registration.processor.reprocess.restart-trigger-filter are also removed.

Can @Khuddusshariff0022 review this change once ?

Reply:
above mention properties are added at line no:
309  registration.processor.reprocess.restart-from-stage
311  registration.processor.reprocess.restart-trigger-filter